tests/python/gaia-ui-tests/gaiatest/apps/camera/app.py

```diff
@@ -115,6 +152,39 @@ def tap_switch_source(self):
         Wait(self.marionette).until(lambda m: not current_camera_mode == self.camera_mode)
         self.wait_for_capture_ready()
 
+    def tap_switch_to_front_camera(self):
+	self.marionette.find_element(*self._camera_switch_to_front_locator).tap()
+	switch_to_rear_button = Wait(self.marionette).until(expected.element_present(*self._camera_switch_to_rear_locator))
+        Wait(self.marionette).until(expected.element_displayed(switch_to_rear_button))
+        
+    def tap_switch_to_rear_camera(self):
```

Comment:
The following methods have the same logic. I think we can extract this logic in a function:
```python
def tap_switch_to_rear_camera(self):
    self._tap_element_and_wait_for_other_to_be_displayed(self._camera_switch_to_rear_locator, self._camera_switch_to_front_locator)

# And so on, for the other methods

def _tap_element_and_wait_for_other_to_be_displayed(self, selector_to_tap, selector_to_wait):
    self.marionette.find_element(*selector_to_tap).tap()
    Wait(self.marionette).until(expected.element_displayed(
        Wait(self.marionette).until(expected.element_present(*selector_to_wait))))
```